.buildkite/build_apk.sh

```diff
@@ -6,7 +6,7 @@ set -euo pipefail
 yes y | rm -r kolibri-kivy/ || true 2> /dev/null
 git clone https://github.com/learningequality/kolibri-kivy.git
 cd kolibri-kivy
-git checkout v0.1.0-beta1
+git checkout develop
```

Comment:
My reason for not pinning this right now was that my assumption is that there will be rapid iteration on this for 0.11. Once we have stabilized for release again, very happy to pin it.

Reply:
Shouldn't we do rapid iteration on the android repo itself then, rather than a workflow comprising of `push test commit on android repo` -> `trigger a build on kolibri`?

Reply:
Except we have no way of triggering the build from the android repo, which is why having it unpinned in Kolibri is helpful. How do we do iteration on the windows installer repo currently?